op-service/txmgr/rpc.go

```diff
@@ -0,0 +1,88 @@
+package txmgr
+
+import (
+	"context"
+	"math/big"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-service/metrics"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+func NewTxmgrApi(txmgr TxManager, m metrics.RPCMetricer, l log.Logger) rpc.API {
+	return rpc.API{
+		Namespace: "txmgr",
+		Service: &TxmgrApi{
+			mgr: txmgr,
+			m:   m,
+			l:   l,
+		},
+	}
+}
+
+type TxmgrApi struct {
+	mgr TxManager
+	m   metrics.RPCMetricer
+	l   log.Logger
+}
+
+func (t *TxmgrApi) GetMinBaseFee(ctx context.Context) *big.Int {
+	recordDur := t.m.RecordRPCServerRequest("txmgr_getMinBaseFee")
+	defer recordDur()
```

Comment:
It's probably better practice to record this than to not, but is there really anything happening in these getters or setters that could take considerable time? Maybe some of these underlying functions aren't as simple as returning a stored value?

Reply:
I was just following convention from other rpc methods. You are correct that there is nothing happening in these getters/setters that should take considerable time. Happy to remove the `recordDur()` calls if preferred

Reply:
All accesses are simple reads and there's no other read or write locks that can lock for any significant time. So I agree, it wouldn't really make sense to record the duration.

I think we could actually just completely drop these metrics. I don't see right now why we would look at them. E.g. looking at the batcher's other admin rpc (op-batcher/rpc), the `StartBatcher` and `StopBatcher` rpcs also don't record any metrics.

Reply:
Dropped all metrics for the rpc api